packages/manager/src/features/Linodes/LinodeEntityDetailBody.tsx

```diff
@@ -323,6 +328,32 @@ export const LinodeEntityDetailBody = React.memo((props: BodyProps) => {
           </Grid>
         </Grid>
       )}
+      {linodeLkeClusterId && (
+        <Grid
+          sx={{
+            borderTop: `1px solid ${theme.borderColors.borderTable}`,
+            padding: `${theme.spacing(2)} ${theme.spacing(2)}`,
+            [theme.breakpoints.down('md')]: {
+              paddingLeft: 3,
+            },
+          }}
+          container
+          direction="column"
+          spacing={2}
+        >
+          <StyledListItem sx={{ borderRight: 'unset' }}>
+            <StyledLabelBox component="span">LKE Cluster:</StyledLabelBox>{' '}
+            <Link
+              data-testid="assigned-lke-cluster-label"
+              to={`/kubernetes/clusters/${linodeLkeClusterId}`}
+            >
+              {cluster?.label ?? linodeLkeClusterId}
```

Comment:
```suggestion
              {cluster?.label ?? `${linodeLkeClusterId}`}
```
We may want to make this change (to ensure fallback is a string instead of a number) to fix this issue: 
![Screenshot 2025-01-27 at 3 40 36 PM](https://github.com/user-attachments/assets/011a079d-0b9c-4c3c-890a-89dff821ddd2)